frontend/src/shared/helpers.tsx

```diff
@@ -952,6 +985,12 @@ export function cohortSampleFilterWhereVariables(
       {
         costCenter_CONTAINS: parsedSearchVals[0],
       },
+      {
+        custodianInformation_CONTAINS: parsedSearchVals[0],
+      },
+      {
+        accessLevel_CONTAINS: parsedSearchVals[0],
+      },
```

Comment:
Realized these fields were never included in the search functionality for Tempo fields, so I added them.